scopes/generator/generator/templates/component-generator/index.ts

```diff
@@ -1,31 +1,41 @@
 import { ComponentContext, ComponentTemplate } from '@teambit/generator';
 import { indexFile } from './files/index';
-import { aspectFile } from './files/aspect-file';
-import { docsFile } from './files/docs-file';
-import { mainRuntime } from './files/main-runtime';
+import { generateTestFileContent } from './files/component-template-files/test';
+import { generateComponentFileContent } from './files/component-template-files/component';
+import { generateCompositionFileContent } from './files/component-template-files/composition';
+import { generateIndexFileContent } from './files/component-template-files/index-file';
+import { componentTemplate } from './files/component-template';
 
 export const componentGeneratorTemplate: ComponentTemplate = {
   name: 'component-generator',
-  description:
-    'create your own component generator \nDocs: https://bit.dev/docs/dev-services/generator/generate-component',
+  description: 'create your own component generator \nDocs: https://bit.dev/reference/generator/create-generator',
   generateFiles: (context: ComponentContext) => {
     return [
       {
         relativePath: 'index.ts',
         content: indexFile(context),
         isMain: true,
       },
+
+      {
+        relativePath: `./${context.name}.ts`,
+        content: componentTemplate(context),
+      },
+      {
+        relativePath: `./files/test-file.ts`,
+        content: generateTestFileContent(),
+      },
       {
-        relativePath: `${context.name}.aspect.ts`,
-        content: aspectFile(context),
+        relativePath: `./files/composition-file.ts`,
+        content: generateCompositionFileContent(),
       },
       {
-        relativePath: `${context.name}.docs.mdx`,
-        content: docsFile(),
```

Comment:
What about the docs?

Reply:
@GiladShoham I followed the pattern we have in our component generator for React